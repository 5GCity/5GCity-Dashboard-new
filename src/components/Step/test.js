/**
 * Steps Component Tests
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import renderer from 'react-test-renderer'
import Steps from './index'
import 'jest-styled-components'

it('renders correctly', () => {
  const tree = renderer
    .create(<Steps />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
