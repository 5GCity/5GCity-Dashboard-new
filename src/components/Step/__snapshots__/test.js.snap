// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background-color: #324148;
}

<div
  className="c0"
/>
`;
